tgui/packages/tgui/interfaces/VotePanel.js

```diff
@@ -22,6 +22,7 @@ export const VotePanel = (props, context) => {
           {choices.map(choice => (
             <Box key={choice}>
               <Button
+                className="Button--multiLine"
```

Comment:
Can you make multiline classify as one of the button's args, then just handle the classes in the button's classes([]) call?
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/4cfc8321-cfc6-497a-a543-11e18843c173)
Similar to how fluid is done here

Reply:
Implemented this and tested it in the latest commit.